### DISCUSSION TITLE
Is there a way to retrieve filteredItems after a search?

### DISCUSSION THREAD

**Discussion by phstc** [pacocoursey/cmdk#187] [General]
Is there a way to retrieve the filter items after searching using the default search?

I have a shortcut cmd+[1-9] for the first nine items from the list. I want to get the filtered items to know which item is when I receive cmd+ N. 

![image](https://github.com/pacocoursey/cmdk/assets/105652/a9d5b8f2-2908-4682-a506-1512b949b63e)

**Comment by edward-meister** [comment]
any idea for this?
